apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-i18n-example.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fdp-list-with-i18n-example',
+    templateUrl: './platform-list-with-i18n-example.component.html'
+})
+export class PlatformListWithi18nExampleComponent {
+    items: any[] = [
+        { 'title': 'objet 1' },
```

Comment:
on page reload, focus is going to first item of this list, why? it should go to the first section, not the i18n section, right?

Reply:
Sure, will look into it

Reply:
Done